File: src/main/java/org/hazelcast/cache/ImperativeToReactiveApplication.java
package org.hazelcast.cache;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories;
import reactor.blockhound.BlockHound;

@SpringBootApplication
@EnableR2dbcRepositories
public class ImperativeToReactiveApplication {

    public static void main(String[] args) {
        BlockHound.install();
        SpringApplication.run(ImperativeToReactiveApplication.class, args);
    }
}

File: src/main/java/org/hazelcast/cache/ImperativeToReactiveApplication.kt
package org.hazelcast.cache

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.r2dbc.repository.config.EnableR2dbcRepositories


@SpringBootApplication
@EnableR2dbcRepositories
class ImperativeToReactiveApplication

fun main() {
    SpringApplication.run(ImperativeToReactiveApplication::class.java)
}
